### ISSUE TITLE
[BUG]  Soroban is not installed!

### ISSUE THREAD

**Issue by MatejMecka** [tolgayayci/sora#19]
**Describe the bug**
Trying to open Sora, but i get Soroban is not installed. Performing soroban from the terminal finds the command and runs it

**To Reproduce**
1. Open the App
2. Soroban is not installed!
You need to install Soroban to use this application. Please visit the repository for more information

**Expected behavior**
See the App

**Screenshots**
<img width="706" alt="Screenshot 2024-07-16 at 20 42 24" src="https://github.com/user-attachments/assets/5ebf1cbc-19b1-4862-a6c7-f8d92472b7ad">

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] MacOS 14.0
 - Soroban Version: 
 
 ```
soroban -V
stellar 21.1.1 (440788e99f3599b8cc9dc0c625fd4869493382c9)
soroban-env 21.1.2 (e0c48cbe91e27bda567767be0f5a5f7fedfa3d77)
soroban-env interface version 90194313216
stellar-xdr 21.1.0 (4dcbb918edcd793fdf064b3ee28ceacc6bd9d1bb)
xdr curr (70180d5e8d9caee9e8645ed8a38c36a8cf403cd9)
```
 - App Version: [e.g. 0.1.0]: v0.2.0

**Additional context**

**Comment by tolgayayci** [comment]
Hi @MatejMecka, thanks for trying out Sora!

I am aware of this bug, Sora now supports `soroban v20.3.4` as latest because soraban-cli is frequently changing at most every 2 months. When we started developing Sora v2, it was still called soroban, but now its name and commands have changed to stellar, and new subcommands have arrived.

Unfortunately, it is difficult to manage changes between versions at this stage because this requires a lot of workload. For example, a command used in the previous version may take on a completely different usage in the new version, which makes compatibility between versions difficult.

Sora will be revised for `stellar v21.0.0` or higher versions, I will write the descriptions more clear on modal for this kind of situations. 

You can try with `soroban v20.3.4` as latest right now, I will ping you when it is working with `stellar v21.0.x` but this will take some time, thanks for your understanding 👍

**Comment by silence48** [comment]
> Hi @MatejMecka, thanks for trying out Sora!
> 
> I am aware of this bug, Sora now supports `soroban v20.3.4` as latest because soraban-cli is frequently changing at most every 2 months. When we started developing Sora v2, it was still called soroban, but now its name and commands have changed to stellar, and new subcommands have arrived.
> 
> Unfortunately, it is difficult to manage changes between versions at this stage because this requires a lot of workload. For example, a command used in the previous version may take on a completely different usage in the new version, which makes compatibility between versions difficult.
> 
> Sora will be revised for `stellar v21.0.0` or higher versions, I will write the descriptions more clear on modal for this kind of situations.
> 
> You can try with `soroban v20.3.4` as latest right now, I will ping you when it is working with `stellar v21.0.x` but this will take some time, thanks for your understanding 👍

I'd suggest not using a child process at all then this problem would never be an issue. Personally i think It's okay for prototyping but usually wouldn't do that for a production application. Version 21 was released April 30th about 2 weeks after 20.3.4 and was still called soroban through all 21 beta and rc releases. It then was changed to stellar. Many of thr commands or options that are changed also still exist such as keys and source.
An idea I had was to make a symlink from stellar to soroban so you can easily see what's broken or just update the process name with a new build.
 Anyways wish you good luck. If you need any support feel free to reach out on discord to me.

**Comment by tolgayayci** [comment]
Hey @silence48,

I am not sure what you suggest replacing to `child_process` but will contact you on discord to discuss. Thanks for suggestion 🥇

**Comment by tolgayayci** [comment]
This issue solved with [Sora v0.2.1](https://github.com/tolgayayci/sora/releases/tag/v0.2.1). Now you can use both for `soroban-cli` and `stellar-cli`. 

In future, we will just support `stellar-cli`, happy using of Sora!
